### PR TITLE
Small update to the Text.rich API doc

### PR DESCRIPTION
**Pull request by HansMuller**
Per https://github.com/flutter/flutter/pull/18635#issuecomment-398796853

### CODE REVIEW

File: packages/flutter/lib/src/widgets/text.dart [hunk 1]
```diff
@@ -179,6 +175,25 @@ class DefaultTextStyle extends InheritedWidget {
 /// )
 /// ```
 ///
+/// Using the [Text.rich] constructor, the [Text] widget can can
```
Comment (goderbauer) [inline packages/flutter/lib/src/widgets/text.dart]: remove one can?

**Review by goderbauer — APPROVED** [review]
LGTM after CLA bot is happy.

File: packages/flutter/lib/src/widgets/text.dart [hunk 2]
```diff
@@ -179,6 +175,25 @@ class DefaultTextStyle extends InheritedWidget {
 /// )
 /// ```
 ///
+/// Using the [Text.rich] constructor, the [Text] widget can
```
Comment (Hixie) [inline packages/flutter/lib/src/widgets/text.dart]: I believe the `new` is supposed to be before the constructor name here (as in `[new Text.rich]`).

**Comment by Hixie** [comment]
@HansMuller What is your plan with this PR? (No particular rush, I'm just cleaning out the review queue.)

**Comment by HansMuller** [comment]
@hixie: I'd forgotten to land it!  Will land it when it's safe to land.

**Pull request merged**